e2e/tests/nightly/gce-to-dataproc.spec.ts

```diff
@@ -72,7 +76,6 @@ describe('Updating runtime compute type', () => {
 
     // Delete runtime
     await notebook.deleteRuntime();
-    await notebook.deleteUnattachedPd();
```

Comment:
`Delete Persistent Disk` button is not found in workspace with CT in staging.

Reply:
Per standup, likely should make this a boolean in workbench-config, with a note to keep synchronized with the enablePersistentDisk feature flag

The alternative would be to read this directly from the API config, but that would be setting a new precedent